internal/ui/table.go

```diff
@@ -448,6 +476,48 @@ func (t *Table) SortColCmd(name string, asc bool) func(evt *tcell.EventKey) *tce
 	}
 }
 
+// SortColChange changes on which column to sort
+func (t *Table) SortColChange(direction SortChange) func(evt *tcell.EventKey) *tcell.EventKey {
+	return func(evt *tcell.EventKey) *tcell.EventKey {
+		sortCol := t.sortCol.Name
+		sortColIdx := -1
+		newSortColIdx := -1
+		prevSortColIdx := -1
+		header := t.GetModel().Peek().GetHeader()
+
+		for i, h := range header {
+			if !t.isVisible(h) {
+				continue
+			}
+
+			if h.Name == sortCol {
```

Comment:
What happens if sortCol is not set? As it stands we would scan the header twice and yield nothing
I think it might be best to list out all visible header columns first. 
Get the current sort column index if set and just go +1 or -1 based on that. 
Bonus: if we are out of range we can do a circular buffer where pressing < on the first col yields the last col and > in the last col bumps you back to the first one.

Reply:
done